### ISSUE TITLE
Hook functions parameters are now typed

### ISSUE THREAD

**Issue by gileri** [wltb/ff_instagram#13]
For example [upstream](https://tt-rss.org/ttrss-docs/classes/Plugin.html#method_hook_subscribe_feed), and [here it isn't](https://github.com/wltb/ff_instagram/blob/215d24f88acb1896c3efc46078e1af37fc33eeb6/init.php#L144).

**Comment by wltb** [comment]
Yes. The entire code should be made PHP 8 compliant as well as it relies heavily on implicit nullification for uninitialized variables.

I hope I'll find some time in the near future to revamp the code entirely and rely on a third party package for scraping. No ETA as yet.
